kvm/requests.yaml

```diff
@@ -5,15 +5,18 @@ releases:
           version: ">= 1.0.1"
           issue: https://github.com/giantswarm/giantswarm/issues/14205
         - name: cluster-operator
-          version: ">= 0.24.2"
+          version: ">= 0.24.2 < 0.25.0"
```

Comment:
cluster-operator 0.25.0 has changes for the per cluster app-operator (v4.0.0 onwards).

But there is a problem. kvm-operator won't delete the cluster namespace because there is the app-operator pod. 🤦‍♂️ 

Will need to do another patch release of the legacy cluster-operator to delete the app CRs. On other providers they are deleted with the namespace.